components/Shared/Input/Number.jsx

```diff
@@ -16,9 +17,50 @@ export const RawNumberInput = styled(BaseInput).attrs(props => ({ ...props }))`
   -moz-appearance: textfield;
 `
 
+const Tag = styled(Box).attrs(props => ({
+  minWidth: 6,
+  bg: 'core.primary',
+  color: 'core.white',
+  p: 2,
+  ...props
+}))`
+  text-align: center;
+  position: absolute;
+  border-radius: 4px;
+  ${color} 
+  ${space} 
+  ${layout}
+  ${border}
+  ${flexbox};
+`
+
+export const DenomTag = props => (
+  <Box
```

Comment:
For consistency in rendering the DenomTag, we wrap it in a relative parent per your suggestion